docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
+- HTTP Archive (sql, csv)
```

Comment:
It _is_ weird to say that HA archives data on HA. But it's an important piece to distinguish, as the BQ requests/pages tables depend on the CSV data archived here.